### DISCUSSION TITLE
Body content doesn't load after adding swiper (svelte) to Electron app

### DISCUSSION THREAD

**Discussion by AsmedianV** [nolimits4web/swiper#4999] [Q&A]
Hello, i have been suffering for several days with 1 problem and without a chance to figure it out on my own.
I am creating an Electron application written in Svelte. Everything works perfectly. But as soon as i add 1 line "import swiper" (slider module) - body content is not loaded in svelte compilation. If I do a window refresh (ctrl+r) - content is loaded.. 
[Video download (1.5mb)](https://cdn.discordapp.com/attachments/892029964034113577/892045044817752104/test-project.mp4)

Please help find and fix problem.

**Comment by nolimits4web** [comment]
Why not checking dev tools console? I am sure there could be some useful log message

**Reply by AsmedianV** [reply]
There are no messages in dev tools. Sorry, I forgot to add this in the description.
I'm 90% sure that the problem is in some kind of conflict with the Rollup plugins, or some additional setting in the config is needed.
